packages/web-runtime/tests/unit/components/Modals/AppTokenModal.spec.ts

```diff
@@ -69,6 +82,17 @@ describe('AppTokenModal component', () => {
   })
 })
 
+const emitNoteInput = (wrapper: VueWrapper<any>, note: string) => {
```

Comment:
I guess this is of type `AppTokenModal.vm`?

```suggestion
const emitNoteInput = (wrapper: VueWrapper<typeof AppTokenModal.vm>, note: string) => {
```

Reply:
Noice, that did the trick. Thank you!